xml/2018-02-05.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<email title="Weekly Digest">
+  <image width="70"
+    src="https://i.imgur.com/L2vMIuM.png"
+    alt="Pie Chart of Last Week's Biscuit Vote"
+    title="Thanks for voting, people who care about the biscuit &lt;3!" />
+
+  <hr />
+
+  <text>
+    Hello everyone,
+
+    Welcome to another week of the DoCSoc digest! It's even more-on-time this
+    week!
+
+    This week, we bring you a Microsoft Cognitive Services workshop!
+
+    **Also, you can sign up for our very own Valentine's Service, only until
+    Thursday 8th February. If here is someone special who you would like to send
+    a rose and message to,
+    [sign up here](https://docs.google.com/forms/d/e/1FAIpQLSdeGAXfxl-DBHFd1xDdP3vMIRcP7oEWZWgJWSYt51-MdayGgw/viewform).
+
+    Last week, over 70 of you voted and decided the future of the biscuit.
+    From now on, the bottom of the digest will include a link to a form where
+    biscuits for the next week may be suggested. The biscuit that we send out
+    remains at the discretion of the DoCSoc committee, but your suggestions
+    should really help. Plus, we'll credit you for the suggestion next to the
+    biscuit form link!
+  </text>
+
+  <hr />
+
+  <heading>Microsoft Cognitive Services: Hands On Workshop</heading>
+  <event>
+    <facebook id="451262598634020" />
+    <text>
+      Come along to learn about Microsoft Cognitive Services, including the
+      Custom Vision AI Service. After learning about these services, there will
+      be a hands on workshop so you can see for yourself how easy it is to build
+      and deploy your projects using these services!
+
+      All experience levels welcome and please make sure you bring your laptop
+      with you. Food and drink will be provided at the event and there will also
+      be giveaways!
+    </text>
+  </event>
+
+  <heading>The Valentine's Bar Night</heading>
+  <event>
+    <facebook id="1607380125981757" />
+    <text>
+      It's time for the first Bar Night of 2018! Once again, we'll be meeting in
+      Metric at 7pm for an evening of free drinks and great conversation.
+
+      Drinks tokens will be given upon entry for FREE to any member of DoCSoc.
```

Comment:
maybe mention the "if you've joined by X date"

Reply:
It's on the fb event and 99% of the ppl reading it are long-time members so I don't think it's necessary. But happy to reconsider if nobody else agrees with me